src/frontend/screens/Settings/CreateTestData.tsx

```diff
@@ -0,0 +1,305 @@
+import {Preset} from '@mapeo/schema';
+import {useMutation, useQueryClient} from '@tanstack/react-query';
+import {lengthToDegrees} from '@turf/helpers';
+import {randomPosition} from '@turf/random';
+import {LocationObject} from 'expo-location';
+import {type BBox} from 'geojson';
+import React, {forwardRef} from 'react';
+import {Controller, useForm} from 'react-hook-form';
+import {StyleSheet, TextInput, ToastAndroid, View} from 'react-native';
+import {UIActivityIndicator} from 'react-native-indicators';
+
+import {useActiveProject} from '../../contexts/ActiveProjectContext';
+import {useDeviceInfo} from '../../hooks/server/deviceInfo';
+import {OBSERVATION_KEY} from '../../hooks/server/observations';
+import {usePresetsQuery} from '../../hooks/server/presets';
+import {useLocation} from '../../hooks/useLocation';
+import {LIGHT_GREY, RED, WHITE} from '../../lib/styles';
+import {Button} from '../../sharedComponents/Button';
+import {LocationView} from '../../sharedComponents/Editor/LocationView';
```

Comment:
not ideal but thought it would be useful to have this info on this screen and didn't want to re-implement this